lib/cdo/graphics/certificate_image.rb

```diff
@@ -4,6 +4,10 @@
 require 'honeybadger/ruby'
 require 'rmagick'
 require_relative '../script_constants'
+require 'cdo/pegasus/donor'
+
+# needed for force_8859_to_utf8
+require 'cdo/pegasus/string'
```

Comment:
For my education, why do we need new requires here if we're just moving the test file?

Reply:
moving the test file exposed that this file had some implicit dependencies. this makes those dependencies explicit.

I believe these dependencies were being provided to the pegasus unit test via the following sequence of requires:
* https://github.com/code-dot-org/code-dot-org/blob/f851edce9c0892b3fd004a8756870956ae8327fc/pegasus/test/test_certificate_image.rb#L2
* https://github.com/code-dot-org/code-dot-org/blob/4c77b30bc0e210e18745070507d5f9df3728a591/pegasus/test/fixtures/mock_pegasus.rb#L1-L0
* https://github.com/code-dot-org/code-dot-org/blob/fc1ff682462f51ddf1e6d6702f1c8c3c3dda432b/pegasus/router.rb#L1
* https://github.com/code-dot-org/code-dot-org/blob/364ca1b25b10ce4d0a2e8379156de02672bbe39d/pegasus/src/env.rb#L2

pegasus server also gets these dependencies via router.rb. I'm not sure where dashboard server is requiring anything in cdo/pegasus, but I assume that it must be happening somewhere because the codepaths in certificate_image.rb have been working in the new dashboard /certificate_images routes.

Reply:
Thanks for the detailed response!